packages/generator-langium/src/index.ts

```diff
@@ -135,6 +140,17 @@ export class LangiumGenerator extends Generator {
                 message: 'Include VSCode extension?',
                 default: 'yes'
             },
+            {
+                type: 'confirm',
+                name: 'includeIntelliJ',
+                // Currently, the IntelliJ plugin depends on files generated for the VS Code extension.
+                when: (answers: Answers): boolean => answers.includeVSCode,
+                prefix: description(
+                    'Your language can be run inside of a JetBrains IntelliJ-based IDE plugin.'
+                ),
+                message: 'Include IntelliJ integration?',
+                default: 'yes',
```

Comment:
Should I keep this default as `yes` or does it make more sense to not include it by default?